friendshipper/src/routes/source/submit/+page.svelte

```diff
@@ -363,6 +364,34 @@
 		return 'bg-primary-500 dark:bg-primary-500';
 	};
 
+	const refreshLocks = async () => {
+		loading = true;
+		try {
+			repoStatus.set(await getRepoStatus());
+		} catch (e) {
+			await emit('error', e);
+		}
+		loading = false;
+	};
+
+	const handleLockSelected = async () => {
+		loading = true;
+		try {
+			if (await acquireLocks($selectedFiles, false)) {
```

Comment:
do we ever use "true" here? If not, i wonder if the boolean return from acquireLocks is necessary.

Reply:
not for now so I can definitely leave it out and add it back in when we actually do need it

Reply:
this `handleLockSelected` will only be used by this component so i think that makes sense :+1: